samples/modules/lvgl/demos/README.rst

```diff
@@ -59,3 +71,11 @@ These demos can be built as follows:
    :gen-args: -DCONFIG_LV_Z_DEMO_WIDGETS=y
    :goals: run
    :compact:
+
+Alternatively, if building from a 64-bit host machine, the previous target
+board argument may also be replaced by ``native_sim_64``.
+
+References
+**********
+
+.. _LVGL demos Readme: https://github.com/lvgl/lvgl/blob/master/demos/README.md
```

Comment:
I think we should rather link to the document within our fork instead of upstream.